factory/tools/cat_MasterLog.py

```diff
@@ -3,9 +3,9 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
-""" Print out the MasterLog for a glidein output file
+"""Print out the MasterLog for a glidein output file
 
- Usage: cat_MasterLog.py logname
+Usage: cat_MasterLog.py logname
```

Comment:
The format for the `Usage` section in this file seems different than the `Usage` section in the next file in this PR. Should we maintain the indentation for consistency?